kyuubi-common/src/test/scala/org/apache/kyuubi/util/KyuubiHadoopUtilsSuite.scala

```diff
@@ -35,4 +37,12 @@ class KyuubiHadoopUtilsSuite extends KyuubiFunSuite {
     assert(hadoopConf.get(xyz) === "abc")
     assert(hadoopConf.get(test) === "t")
   }
+
+  test("encode/decode Writable object") {
+    val writable = new Text("Hello, World!")
+    val decoded = new Text()
+
+    KyuubiHadoopUtils.decodeWritable(decoded, KyuubiHadoopUtils.encodeWritable(writable))
```

Comment:
why doesn't `decodeWritable` named decodeString and return `Text`?

Reply:
The methods naming and implements are from `org.apache.hadoop.security.token.Token#encodeWritable` and `org.apache.hadoop.security.token.Token#decodeWritable` and  should be able to convert any `Writable` object. 
Here, `encodeWritable` is used to convert Credentials to String at Kyuubi Server side and `decodeWritable` is used to convert String to Credentials at Kyuuib SQL engine side.
Change `decodeWritable` to `decodeString` and return `Text` maybe is not proper  for its usage.